charts/tidb-cluster/values.yaml

```diff
@@ -48,9 +48,20 @@ discovery:
 enableConfigMapRollout: false
 
 pd:
+  # Please refer to https://github.com/pingcap/pd/blob/master/conf/config.toml for the default
+  # pd configurations (change to the tags of your pd version), 
+  # just follow the format in the file and configure in the 'config' section
+  # as below if you want to customize any configuration.
+  # Please refer to https://pingcap.com/docs-cn/v3.0/reference/configuration/pd-server/configuration-file/
+  # (choose the version matching your pd) for detailed explanation of each parameter.
+  config: |
+    [log]
+    level = "info"
+    [replication]
+    location-labels = ["region", "zone", "rack", "host"]
```

Comment:
It should be reminded that `location-labels` should be consistent with `tikv.storeLabels`.

Reply:
Reminder was added in the tikv.storeLabels section.